packages/core/src/shared/extensionStartup.ts

```diff
@@ -49,6 +49,7 @@ export async function maybeShowMinVscodeWarning(minVscode: string) {
     }
 }
 
+// TODO: Delete? as it was only used in Cloud9
```

Comment:
yeah, we dropped the "quickstart" concept, which had overlap with (1) the marketplace readme, and (2) the new notifications / whats-new concept.